daemons/copy-offload/cmd/main.go

```diff
@@ -72,6 +73,26 @@ func setupLog() logr.Logger {
 func setupClient(crLog logr.Logger) client.Client {
 	config := ctrl.GetConfigOrDie()
 
+	qpsString, found := os.LookupEnv("NNF_REST_CONFIG_QPS")
+	if found {
+		qps, err := strconv.ParseFloat(qpsString, 32)
+		if err != nil {
+			crLog.Error(err, "invalid value for NNF_REST_CONFIG_QPS")
+			os.Exit(1)
+		}
+		config.QPS = float32(qps)
```

Comment:
Should we default these to something better than config's defaults of 5 and 10?

Reply:
If we do, let's put it in the default copy-offload container profile.
What numbers should I pick?

Reply:
Done